### ISSUE TITLE
Get vips xoffset/yoffset from image

### ISSUE THREAD

**Issue by AndreKR** [lovell/sharp#3780]
### What are you trying to achieve?

I'm trying to align multiple texts at their baselines. Some cursory experimentation with pyvips suggest that it _might_ be possible to do this if [`yoffset`](https://www.libvips.org/API/current/VipsImage.html#VipsImage--yoffset) was available.

In sharp it is not:

```javascript
let sharp = require('sharp');

(async function () {
	let image = sharp({
		text: {
		  text: 'Hello'
		}
	});

	console.log(await image.metadata());
})();
```
```json5
{
  format: 'raw',       
  width: 30,
  height: 10,
  space: 'b-w',        
  channels: 1,
  depth: 'uchar',      
  density: 72,
  isProgressive: false,
  hasProfile: false,   
  hasAlpha: false      
}
```

### When you searched for similar feature requests, what did you find that might be related?

"yoffset" appeared in several issue but none were _about_ `yoffset`.

### What would you expect the API to look like?

`(await image.metadata()).yoffset`

### What alternatives have you considered?

-

### Please provide sample image(s) that help explain this feature

-

**Comment by lovell** [comment]
Yes, this would make a useful enhancement as libvips calculates the top-left position but this is not yet exposed by sharp. A possible implementation might look something like the following:

```js
// PROPOSED API - NOT YET AVAILABLE
const { info } = await sharp({ text: { ... }).toBuffer({ resolveWithObject: true });
const { textOffsetLeft, textOffsetTop } = info;
```
